docs/parsec_user_guide.md

```diff
@@ -27,6 +27,7 @@ If you are using MacOS or an Ubuntu-like distribution, you can run the system wi
 
 Build and run the system:
 ```console
+$ ./scripts/configure.sh
```

Comment:
I believe we didn't include this because it's included in the setup instructions in the top level README.md, but I have no problem including it -- I would just make a point that configure is only required once if none of the associated packages change

Reply:
Good point - addressed by adding a comment to the line.